### DISCUSSION TITLE
Web3Auth Node appears down

### DISCUSSION THREAD

**Discussion by davedefiq** [Web3Auth/Web3Auth#1129] [Help]
We've been experiencing several users unable to reconstruct their wallets. We use CustomAuth and we've tracked down the potential root cause being this node returning 502 Bad Gateway:

https://torus-node.ens.domains/jrpc

Is this issue known, and will just 1 node being unresponsive cause wallet retrieval failures?

**Comment by shahbaz17** [comment]
Hey @davedefiq 

Thanks for reporting. We are working on it. It should not affect the wallet retrieval as 5 out of 9 nodes are enough to retrieve it. Please ignore this or remove this ens from the list to fix it in the interim. The fix is coming soon.

**Comment by chaitanyapotti** [comment]
This is a problem with android sdks only for customauth.
This is fixed now and new versions have been released. Pls upgrade to the latest

**Reply by acolls-g4al** [reply]
also with node/sdk mainnet

**Reply by vukhaihoan** [reply]
customAuth react native ( Ios ) not fixed

**Comment by tranbathanhtung** [comment]
@chaitanyapotti I got the same issue on web SDK ... I upgrade the SDK to the latest version ... Can you check it again?

```
    "@web3auth/base": "^4.0.0",
    "@web3auth/core": "^4.0.0",
    "@web3auth/openlogin-adapter": "^4.1.0",
```

<img width="1239" alt="image (1)" src="https://user-images.githubusercontent.com/26837622/209755791-1eeaa0d1-8d99-4212-82da-af3fc88b1613.png">

![image](https://user-images.githubusercontent.com/26837622/209755701-81ffd78e-4b8c-4a3f-9183-f34ff1c0e888.png)
